src/scripts/main.js

```diff
@@ -1 +1,9 @@
 'use strict';
+
+const imageToChange = document.querySelector('#largeImg');
+const ul = document.querySelector('.gallery__list');
```

Comment:
```suggestion
const list = document.querySelector('.gallery__list');
```
Better not to give names of tags to your variables